### PR TITLE
a typo in english blog tutorial and some turkish files.

### PR DESCRIPTION
**Pull request by mtkocak**

### CODE REVIEW

File: tr/console-and-shells.rst [hunk 1]
```diff
@@ -0,0 +1,946 @@
...
+    sisteminizde mevcut olmalıdır.
+
+Detaylara geçmeden önce, CakePHP konsolunu çalıştırabildiğinizden emin olalım.
+Önce bir sistem kabuğu getirmeniz gerekir. Bu bölümdeki örnekler bash üzerinde
+gösterilecekler, ancak CakePHP konsolu Windows ile de uyumludur.
+Bu örnek, kullanıcının bir bash istemcisinde oturumu olduğunu ve CakePHP
+uygulamasının kök dizininde olduğunu varsaymaktadır.
+
+CakePHP uygulamaları bir uygulama için tüm kabuk ve görevleri içeren ``Console``
+dizini içerirler. Ayrıca çalıştırılabilir uygulama da mevcuttur:
```
Comment (markstory) [inline tr/console-and-shells.rst]: Should end in `::` that triggers formatted blocks in sphinx.

Comment (mtkocak) [inline tr/console-and-shells.rst]: which line? And how should I change that?

Comment (antograssiot) [inline tr/console-and-shells.rst]: line 40 should end with :: instead of :

File: en/tutorials-and-examples/blog/part-three.rst [hunk 2]
```diff
@@ -177,7 +177,7 @@ read if you want re-familiarize yourself with how CakePHP works.
 .. note::
     If you are on Windows remember to use \ instead of /.
 
-You'll need to edit the following in **src/Template/Categories/index.ctp**::
+You'll need to edit the following in **src/Template/Categories/add.ctp**::
```
Comment (cake17) [inline en/tutorials-and-examples/blog/part-three.rst]: This has already been changed in #3160. Can you cancel the changes for this file ?

**Comment by cake17** [comment]
Nice work @mtkocak  :+1: . I can take care of creating the structure with the empty contributing files.

@mtkocak would it be possible to have the following message translated in turkish:

    The documentation is not currently supported in Turkish language for this page.

    Please feel free to send us a pull request on Github or use the Improve This Doc button to directly propose your changes.

    You can referer to the english version in the select top menu to have information about this page’s topic.

This will be the default message when pages are not translated, so it's better if it's directly in turkish

File: en/tutorials-and-examples/blog/part-three.rst (hunk 2 above)
Comment (mtkocak) [inline en/tutorials-and-examples/blog/part-three.rst]: ok

**Comment by mtkocak** [comment]
@cake17 

    Bu belge şu anda bu sayfa için Türkçe dilinde desteklenmemektedir.

    Yapmak istediğiniz değişiklikleri göndermek için, Github üzerinden istek yollaryın ya da bu sayfanın üzerinde bulunan "Improve This Doc" butonuna tıklamaktan lütfen çekinmeyin.
    
    Bu sayfanın konusu hakkındaki İngilizce kaynağa erişmek için üst taraftaki seçim menüsünü kullanabilirsiniz.

File: tr/console-and-shells.rst (hunk 1 above)
Comment (mtkocak) [inline tr/console-and-shells.rst]: I think I resolved it.

**Comment by mtkocak** [comment]
@cake17 By the way, if you add empty files, it is going to be easier to translate for me.

File: en/tutorials-and-examples/blog/part-three.rst [hunk 3]
```diff
@@ -176,8 +176,12 @@ read if you want re-familiarize yourself with how CakePHP works.
 .. note::
     If you are on Windows remember to use \ instead of /.
 
+<<<<<<< HEAD
+You'll need to edit the following in **src/Template/Categories/add.ctp**::
```
Comment (antograssiot) [inline en/tutorials-and-examples/blog/part-three.rst]: @mtkocak You need to solve this conflict so your PR will be mergeable again

**Comment by cake17** [comment]
@mtkocak as soon as this PR can be merged, I create the tree structure with empty files.

File: en/tutorials-and-examples/blog/part-three.rst (hunk 3 above)
Comment (cake17) [inline en/tutorials-and-examples/blog/part-three.rst]: @mtkocak You can just keep

    You'll need to edit the following in **src/Template/Categories/add.ctp**
    and **src/Template/Categories/edit.ctp**::

**Comment by cake17** [comment]
@antograssiot I merge this and I will make the changes

**Comment by cake17** [comment]
thanks @mtkocak !

**Pull request merged**